### ISSUE TITLE
Retention Creation Confusion

### ISSUE THREAD

**Issue by rconiv** [MicrosoftDocs/microsoft-365-docs#5756]
I created a ticket about this, and talked to a support tech (Case# 26731025).

If I go in to information governance, retention, and create a retention policy, and assign a mailbox to it for testing, after enabling archiving, I don't see a message in the inbox stating it has a policy setup on it (did the generic retain for 7 years, and then delete).  When I deleted the account, it did not show up as an inactive mailbox.

If I go to classic exchange after enabling archiving, and enable an old MRM based retention policy, it shows up in outlook, and when I delete the mailbox, it shows up in the inactive mailbox.  

Basically my issue is none of the retention documents I have found have worked, and even state I should not use the MRM anymore, but I have found that retention doesn't work unless I do, and the tech even said I should still keep using classic exchange MRM for now, and that at some point the features there will be migrated to the modern exchange interface.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ef663fdc-1776-3c73-7743-332796c284e6
* Version Independent ID: ceb29788-39be-8c30-b92a-7a22243ebe83
* Content: [Create and configure retention policies to automatically retain or delete content - Microsoft 365 Compliance](https://docs.microsoft.com/en-us/microsoft-365/compliance/create-retention-policies?view=o365-worldwide)
* Content Source: [microsoft-365/compliance/create-retention-policies.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/compliance/create-retention-policies.md)
* Service: **o365-seccomp**
* GitHub Login: @cabailey
* Microsoft Alias: **cabailey**

**Comment by cabailey** [comment]
Looking into this

**Comment by rconiv** [comment]
@yogkumgit what writer-input-required are you all needing, to warrant the tag?  I haven't seen any questions, just @cabailey  looking in to it 5 days ago.

**Comment by cabailey** [comment]
@rconiv  Thanks for your patience while I reviewed the case notes and made inquiries from the support team.  There are just 2 scenarios where you might need to continue to use the older retention policies: For an archiving policy (applicable to you) to move emails from the primary mailbox to the archive mailbox, and to automatically apply different retention settings to different Outlook folders (instead, users can achieve the same result by applying a default retention label).   If you need to continue to use the older retention policies for these scenarios, they can work side by side with Microsoft 365 retention policies and retention labels. 

We'll add these exceptions to the documentation to clarify when you might need to keep using them.  But for the stated business requirement I see in the support notes (for eDiscovery purposes, retain email for 7 years and then permanently delete it), a Microsoft 365 retention policy is recommended and should work. I don't see anything in the notes that suggest it wouldn't work and you should exclusively use the old retention policies?

**Comment by rconiv** [comment]
Thanks, what we are needing it for is when someone leaves the company, and I delete their M365 Office license, the mailbox will go to inactive.  If I enable archiving, and have everyone in the company or just the person tied to the policy I created under the M365 Admin\compliance\information governance\retention policies, and delete the account (did on test account), it doesn't show up under inactive inbox in the retention policies area.  It also doesn't show retention is enabled in Outlook 365.  

If though I have the archive enabled, but go in to classic exchange admin center and then go to recipients\mailboxes, and double click on the person while in something that is not Edge Chromium (it crashes a lot as soon as I pick anything in there), and go to mailbox features and under retention pick something besides MRM default, it will show up as a retention policy on the persons inbox and when I delete the account it shows up in the inactive inbox area in retention policies under the modern M365. 

Basically the issue is, is the documentation, and the website I am doing this in, is saying I should use the modern M365 Admin area, but it doesn't seem to actually enable the retention policy created there, or at least what is needed to have a mailbox go to inactive mode when deleted, and is just deleted.

**Comment by cabailey** [comment]
Yes, what you're requesting for an inactive mailbox is documented here: https://docs.microsoft.com/en-us/microsoft-365/compliance/retention-policies-exchange#when-a-user-leaves-the-organization  

"If a user leaves your organization and the user's mailbox is included in a policy for retention, the mailbox becomes an inactive mailbox when the user's Microsoft 365 account is deleted. The contents of an inactive mailbox are still subject to any retention policy that was placed on the mailbox before it was made inactive, and the contents are available to an eDiscovery search. For more information, see Inactive mailboxes in Exchange Online."

It doesn't explicitly include archiving but I don't see why that would make any difference and I don't see this issue being addressed in the support notes.  If the mailbox didn't go inactive for you I'm wondering if it was because of a timing delay.  What did the support engineer advise here?  

Can you also explain what you mean by the retention policy "... doesn't show retention is enabled in Outlook 365".  What were you expecting to see that wasn't there?

**Comment by rconiv** [comment]
I found that if I didn't enable archiving, I could pick either retention option, and and when the mailbox was deleted in M365, it didn't go to inactive mode.  If I did activate it, and enable the retention in classic mode, it would go inactive.  I though could enable archive, and retention in the modern M365, wait a day or 3, and delete the account, and it wouldn't go to inactive mode.

The support engineer basically said I have to enable archiving and do retention in classic mode, because the capability hasn't migrated yet to the modern mode, even though there are banners all over classic mode saying do this in modern mode.  They did though say the modern view retention is either everyone, or have to pick people one at a time.  It was a phone conversation, so I can't copy/paste exactly what they said.

When I enable archiving, and the retention in classic view it would eventually say on emails that they would be deleted in 7 years.   At least there used to be, though I had turned off the retention on my account in classic view at some point, and just turned it back on, so it could take a while to see if it shows up again.

**Comment by cabailey** [comment]
Thanks for the screenshot - yes, that's the same notification on emails I see when I assign Microsoft 365 retention policies to a mailbox, and the experience is documented here: https://docs.microsoft.com/en-us/microsoft-365/compliance/retention-policies-exchange#user-notification-of-expiry-date  If you weren't seeing this for your retention policy, it might be a timing delay and you should ask the support engineer to investigate (I didn't see this in the support case notes). The behavior doesn't change if you enable archiving.

I've verified that the behavior and timing for a mailbox going inactive is also the same for Microsoft 365 retention policies as it is for MRM. I didn't see this addressed in the support notes either, but the first thing to confirm is that the policy is assigned to the mailbox before the account is deleted.

Good news - you don't have to select users one-by-one if you configure a retention policy for Exchange and you use the options to "include" or "exclude". You can select distribution groups or email-enabled security groups and behind the scenes, these groups are automatically expanded to select the mailboxes of the users in the groups. This is a very efficient way to select multiple recipients who are already grouped, and we're soon going to release another feature that is even more flexible - adaptive scopes. Then you'll be able to specify users based on their Azure AD attributes, such as users in a specific region or specified departments. You can also use custom Exchange attributes. It's called adaptive because new users will be picked up (or removed) dynamically. In the meantime, you might want to look into the group configuration, documented here: https://docs.microsoft.com/en-us/microsoft-365/compliance/create-retention-policies#configuration-information-for-exchange-email-and-exchange-public-folders

In summary, we do recommend you use Microsoft 365 retention rather than MRM, even if for now, you need to carry on using MRM for your archiving policy (and the two solutions are designed to co-exist). Microsoft 365 retention has a lot of benefits that you can easily extend to other workloads, with new functionality and enhancements being added all the time. I'll close this GitHub Issue for the documentation now that we clarify the two MRM features that don't have parity: https://docs.microsoft.com/en-us/microsoft-365/compliance/retention#use-retention-policies-and-retention-labels-instead-of-older-features.  

Thank you for your patience in trying the new retention policies, and I encourage you to talk with Support again to investigate the unresolved issues you raised.
